docs/source/features/pre_screening.rst

```diff
@@ -34,6 +34,15 @@ To select a dataset:
 .. figure:: ../../images/asreview_prescreening_datasets.png
    :alt: ASReview dataset selector
 
+.. warning::
+
+    If you upload your own data make sure to remove duplicates and make sure
```

Comment:
Perhaps this sentence can be shortened to:
"If you upload your own data, make sure to remove duplicates and to retrieve as many abstracts as possible (`don't know how (..))."